src/assets/styles/global.scss

```diff
@@ -55,4 +55,7 @@ body {
     .ui.container, .ui.grid.container {
         width: 95% !important;
     }  
-}
\ No newline at end of file
+}
+.ck-rounded-corners .ck.ck-editor__main>.ck-editor__editable, .ck.ck-editor__main>.ck-editor__editable.ck-rounded-corners{
```

Comment:
@Ekta29yadav if you use CKEditor classes it will break other places

Reply:
@pallakartheekreddy this is for fix, if any other place this issue came it works and this fix not break anything